lib/spring/client/run.rb

```diff
@@ -1,3 +1,4 @@
+require "spring/commands"
```

Comment:
@sonalkr132 hey, quick question. I was wondering why you added this line? I'm getting an exception which seems to go away when I remove this line (which I've logged [here](https://github.com/rails/spring/issues/457)).